fairscale/optim/adascale.py

```diff
@@ -26,12 +26,13 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+# type: ignore
```

Comment:
Can you open an issue for making adascale type-checked.

Reply:
Sure. #143 filed. Will get to it after reproducibility work.